_posts/2024-06-30-concurrent-search-follow-up.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: post
+title:  "Exploring concurrent segment search performance"
+authors:
+ - jaydeng
+ - sohami
+date: 2024-06-30
+categories:
+ - search
+ - technical-post
+meta_keywords:
+meta_description:
```

Comment:
Please use this updated meta: 

_meta_keywords:_ `concurrent segment search, search concurrency, control search concurrency, searching segments concurrently in OpenSearch`

_meta_description:_ `Learn how to benchmark, track performance, and improve latency across your large workloads by searching segments concurrently in OpenSearch. `